internal/cli/clients/client.go

```diff
@@ -1461,12 +1474,12 @@ func (c *EpinioClient) logs(appName, org string) (context.CancelFunc, error) {
 		Include:               nil,
 		Timestamps:            false,
 		Since:                 duration.LogHistory(),
-		AllNamespaces:         false,
-		LabelSelector:         labels.Everything(),
+		AllNamespaces:         true,
+		LabelSelector:         selector,
 		TailLines:             nil,
 		Template:              tailer.DefaultSingleNamespaceTemplate(),
-		Namespace:             org,
-		PodQuery:              regexp.MustCompile(fmt.Sprintf(".*-%s-.*", appName)),
+		Namespace:             "",
```

Comment:
Can we not remove the Namespace entry because we scan all?

Reply:
no it's either all or one. It doesn't seem to accept a list of Namespaces